javascript/frameworks/cap/lib/advanced_security/javascript/frameworks/cap/CDS.qll

```diff
@@ -259,6 +274,27 @@ class ServiceInstanceFromServeWithParameter extends ServiceInstance {
   }
 }
 
+abstract class CdsDbService extends ServiceInstance {
+  /* A DB service is implicitly defined. */
+  override UserDefinedApplicationService getDefinition() { none() }
+}
+
+class GloballyAccessedCdsDbService extends CdsDbService {
+  GloballyAccessedCdsDbService() {
+    exists(CdsFacade cds | not cds.isFromCdsLib() |
+      this = cds.getMember("db").asSource() or
+      this = cds.asSource()
+    )
```

Comment:
The logic in this characteristic predicate is quite similar to the `CdsDb` class:
 * Can we combine the two definitions?
 * Are they intended to be different?

Reply:
Addressed in commit [cee86f1](https://github.com/advanced-security/codeql-sap-js/pull/200/commits/cee86f160920c09add5050c8734af799e459de24). The `CdsDb` is now made to absorbed this class.